ui/tests/acceptance/secrets/backend/kv/kv-create-new-version-test.js

```diff
@@ -63,6 +63,6 @@ module('Acceptance | kv | creates a secret and a new version', function (hooks)
       `/vault/secrets/${this.mountPath}/kv/${secretPath}/details?version=2`,
       'list view navigates to latest version'
     );
-    assert.dom(PAGE.detail.versionTooltip).hasTextContaining('Version 2');
+    assert.dom(PAGE.detail.versionCreated).hasTextContaining('Version 2');
```

Comment:
Changed this selector globally to be more indicative of what it is

Reply:
I had changed this to `versionTooltip` since it renders elsewhere for deleted times and updated ties

Reply:
I like versionTimestamp! I'll change it